src/v3/src/transformernew/attribute/transform.test.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Input } from '@okta/okta-auth-js';
+
+import { transformField } from './transform';
+
+describe('Attributes transformer', () => {
+  it('should return uischema object with options.attributes.autocomplete === "username" if formfield.name === "identifier" is present in ion object', () => {
+    const formfield: Input = { name: 'identifier' };
+
+    const result = { attributes: { autocomplete: 'username' } };
+
+    expect(transformField(formfield)).toEqual(result);
+  });
+
+  it('should return uischema object with options.attributes.autocomplete === "current-password" if formfield.name === "password" is present in ion object', () => {
+    const formfield: Input = { name: 'password' };
+
+    const result = { attributes: { autocomplete: 'current-password' } };
+
+    expect(transformField(formfield)).toEqual(result);
+  });
+
+  it('should return uischema object with options.attributes.autocomplete === "current-password" if formfield.name === "newPassword" is present in ion object', () => {
+    const formfield: Input = { name: 'newPassword' };
+
+    const result = { attributes: { autocomplete: 'current-password' } };
```

Comment:
shouldn't autocomplete be `"new-password"`?

Reply:
@glenfannin-okta can you confirm?

Reply:
Actually, this value does not matter, `newPassword` doesnt seem to be an actual value now that we are using generic remediator version of auth-js. Seems to always be `credentials.passcode` no matter the flow. So this line can be removed completely. We actually update the autocomplete value in the custom transformer because there is no way to decipher what it should be at this level.